.travis.yml

```diff
@@ -54,39 +54,39 @@ jobs:
       install:
         - conda install --yes python=2.7 flake8
       script:
-        - if [[ "$LINT" == "1" ]]; then flake8 . --exclude=.venv,.build,planemo_test_env,build --ignore=E501,F403,E402,F999,F405,E712 ; fi
+        - if [[ "$LINT" == "1" ]]; then flake8 . --exclude=.venv,.build,planemo_test_env,build --ignore=E501,F401,F403,E402,F999,F405,E712 ; fi
     - stage: test
       os: linux
       env: TRAVIS_PYTHON_VERSION=2.7
-      script: py.test hicexplorer/test/long_run/test_build_matrix_trivial_runs.py
+      script: py.test hicexplorer/test/long_run/test_hicBuildMatrix_trivial_runs.py
     - stage: test
       os: osx
       env: TRAVIS_PYTHON_VERSION=2.7
-      script: py.test hicexplorer/test/long_run/test_build_matrix_trivial_runs.py
+      script: py.test hicexplorer/test/long_run/test_hicBuildMatrix_trivial_runs.py
     - stage: test
       os: linux
       env: TRAVIS_PYTHON_VERSION=3.6
-      script: py.test hicexplorer/test/long_run/test_build_matrix_trivial_runs.py
+      script: py.test hicexplorer/test/long_run/test_hicBuildMatrix_trivial_runs.py
     - stage: test
       os: osx
       env: TRAVIS_PYTHON_VERSION=3.6
-      script: py.test hicexplorer/test/long_run/test_build_matrix_trivial_runs.py
+      script: py.test hicexplorer/test/long_run/test_hicBuildMatrix_trivial_runs.py
     - stage: test
       os: linux
       env: TRAVIS_PYTHON_VERSION=2.7
-      script: py.test hicexplorer/test/long_run/test_buildMatrix.py
+      script: py.test hicexplorer/test/long_run/test_hicBuildMatrix.py
     - stage: test
       os: osx
       env: TRAVIS_PYTHON_VERSION=2.7
-      script: py.test hicexplorer/test/long_run/test_buildMatrix.py
+      script: py.test hicexplorer/test/long_run/test_hicBuildMatrix.py
     - stage: test
       os: linux
       env: TRAVIS_PYTHON_VERSION=3.6
-      script: py.test hicexplorer/test/long_run/test_buildMatrix.py
+      script: py.test hicexplorer/test/long_run/test_hicBuildMatrix.py
     - stage: test
       os: osx
       env: TRAVIS_PYTHON_VERSION=3.6
-      script: py.test hicexplorer/test/long_run/test_buildMatrix.py
+      script: py.test hicexplorer/test/long_run/test_hicBuildMatrix.py
     - stage: test
```

Comment:
unrelated to the actual PR, but what is the advantage of running each tests separately?

Reply:
With the separate runs the test cases are executed (more or less) in parallel and we fulfill the time limits of travis per job. If we don't do this the test cases would be quitted by travis after 40 - 50 minutes.

Reply:
ahhh!! cool